src/coreComponents/mesh/MeshObjectPath.cpp

```diff
@@ -80,14 +80,15 @@ MeshObjectPath::fillPathTokens( string const & path,
 
   int objectIndex = findObjectIndex();
 
-  GEOSX_ERROR_IF( objectIndex==-1,
-                  GEOSX_FMT( "Path ({}) does not contain a valid object type. "
-                             "Must contain one of ({},{},{},{})",
+  GEOSX_THROW_IF( objectIndex==-1,
+                  GEOSX_FMT( "Path {} does not contain a valid object type. "
```

Comment:
Are you sure it's more readable not to have the paths surrounded by `"` or anything?

Reply:
As there are no spaces in this kind of path, the users feedback is that it doesn't imply any readability issue.
More over I think that's consistent with the other messages using getPath().
Anyway, these paths are quite hard to understand for users. What do you think of outputting the XML hierarchy instead to have a shorter and clearer one? (when applicable)

Reply:
What's most convenient w.r.t. users is really up to you, you can exchange with them better than I can.